src/contexts/tour-context/context.tsx

```diff
@@ -0,0 +1,876 @@
+                            variableRadioOptionDomMask.hideMask()
+                            
+                            const variableViewOption = getVariableViewRadioOption()
+                            variableViewOption?.removeEventListener("click", listener)
+                        },
+                        cancel: () => {
+                            variableRadioOptionDomMask.hideMask()
+                            
+                            const variableViewOption = getVariableViewRadioOption()
+                            variableViewOption?.removeEventListener("click", listener)
+                        },
+                        complete: () => {
+                            variableRadioOptionDomMask.hideMask()
+
+                            const variableViewOption = getVariableViewRadioOption()
+                            variableViewOption?.removeEventListener("click", listener)
+                        }
+                    }
+                })()
+            },
+            {
+                id: "main.search.variable-view.intro",
+                path: searchPathPattern,
+                attachTo: {
+                    element: "head"
+                },
+                title: "Variable search",
+                text: renderToStaticMarkup(
+                    <div>
+                        Variable search returns all the variables with a match with the search term.
+                        HSS provides a histogram filter to zoom in on higher-scoring results (those more
+                        relevant to the search query). Hovering the mouse pointer over each variable in the
+                        histogram displays additional information, including its name, description, and study.
+                    </div>
+                ),
+                buttons: [
+                    {
+                        classes: secondaryButtonClass,
+                        text: 'Back',
+                        action: function() {
+                            const conceptGridViewOption = getConceptViewRadioOption()!
+                            this.back()
+                            conceptGridViewOption.click()
+                        }
+                    },
+                    {
+                        classes: primaryButtonClass,
+                        text: 'Next',
+                        type: 'next'
+                    }
+                ],
+            },
+            {
+                id: "main.search.variable-view.study-list",
+                path: searchPathPattern,
+                attachTo: {
+                    element: studyListItemDomMask.selector,
+                    on: "bottom"
+                },
+                scrollToHandler: () => scrollIntoViewIfNeeded(studyListItemDomMask.originalSelector),
+                text: renderToStaticMarkup(
+                    <div>
+                        You can also click to expand each study to view the study variables associated
```

Comment:
Do we want to highlight that you can now filter variable search results by data source (e.g. "HEAL Studies (284)", "HEAL Research Programs (74)", "Non-HEAL Studies (5441)") in the tour? Maybe in a future release?

Reply:
Yep.